package.json

```diff
@@ -5,13 +5,15 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon index.js",
-    "test": "NODE_ENV=test mocha --require babel-core/register --exit"
+    "test": "cross-env NODE_ENV=test nyc mocha --require babel-core/register --exit",
+    "coverage": "npm test && nyc report --reporter=text-lcov | coveralls"
```

Comment:
Why the `npm test` in the coverage script?

Reply:
I wanted it to run the test before generating the report.
While developing locally we could just run `test run coverage` which will both run the test and code coverage.